src/mp-instance.js

```diff
@@ -223,6 +231,22 @@ export default function mParticleInstance(instanceName) {
         if (queued) return;
         this._Persistence.setDeviceId(guid);
     };
+
+    /**
+     * Sets the Package Name
+     * @method setPackageName
+     * @param {String} name Package Name
+     */
+    this.setPackageName = function(packageName) {
+        var queued = queueIfNotInitialized(function() {
+            self.setPackageName(packageName);
+        }, self);
+
+        if (queued) return;
+
+        self._Store.SDKConfig.packageName = packageName;
+        self._Persistence.update();
```

Comment:
Thinking about this more, I actually don't think we need the setter or getter on the API.  The point of this is to be set on the config, right?  I could see why we'd want a `getter`, but really a packageName shouldn't be changed mid-session.  You probably followed in the `setAppName` footsteps, but I'd argue we should remove that from our next major version and only allow them to do it on the config also since appName won't change mid session.

Reply:
More food for thought.  Does native support either a getter or setter outside of where the bundle id lives and automatically gets read and sent to our servers?  My inclination is no, but we should double check...and if Native doesn't support this (or getter) and only supports automatically reading the bundleid and passing it on, that's even more reason to not support on web.  As a plus, this would help with code bloat too, and simplify this PR!

Reply:
That would mean not referencing `getPackageName` in `serverModel.js` and just grabbing from the `_Store` directly also.

Reply:
I think that makes sense and is more consistent with Native. I'll refactor this.